rspec-rails.gemspec

```diff
@@ -21,7 +22,7 @@ Gem::Specification.new do |s|
     'source_code_uri'   => 'https://github.com/rspec/rspec-rails',
   }
 
-  s.files            = `git ls-files -- lib/*`.split("\n")
+  s.files            = Rake::FileList["lib/**/{.*,*}"].exclude(*File.read(".gitignore").split).reject { |f| File.directory?(f) }
```

Comment:
Does `exclude` make any difference?

Reply:
Not really, just helps to respect the `.gitignore` file.

Reply:
This still doesn't avoid uncommitted files...